sui/Dockerfile

```diff
@@ -10,13 +10,14 @@ WORKDIR /tmp
 
 RUN sui genesis -f
 COPY scripts/start_node.sh  .
-COPY scripts/funder.sh  .
```

Comment:
If we no longer use the faucet, do we manually fund the deployer account before deploying the contracts?